### PR TITLE
fix: allow fixup script to remove bad settings

### LINKED ISSUE #5183: event-broker: "Resource not found" to topic, but topic exists and has not changed
https://sentry.prod.mozaws.net/operations/fxa-event-broker-prod/issues/6712914/events/latest/

It’s started happening when I ran the script. It’s complaining Resource not found (resource=rpQueue-E6EB0D1E856335FC but I’m looking at that topic in the console, and our terraform config says “nothing to do”. i.e., state of config has not changed in prod

One small difference is that the hex is uppercase in the error message, but the topic is lowercase, like all our usage of oauth client id in our code base. So that difference my not be relevant.

A historical sentry entry from also had uppercase hex for rpQueue-DEADBEEF

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-1810)
┆Issue Number: FXA-1810

### PR DESCRIPTION
**Pull request by bbangert**
Because:

* The script was run unexpectedly with an uppercase
  client id, the wrong queue name is tried.

This commit:

* Allows ops to run the script to remove the entries
  made in error.

Closes #5183

### CODE REVIEW

File: packages/fxa-auth-server/bin/update_firestore_from_oauth.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const log = require('../lib/log')(
 );
 
 const CLIENT_ID = process.env.CLIENT_ID;
```
Comment (jrgm) [inline packages/fxa-auth-server/bin/update_firestore_from_oauth.js:23]: Can we add an assertion here that the client has the proper case. Abort on `/[A-Z]/.test(CLIENT_ID)`?

File: packages/fxa-auth-server/bin/update_firestore_from_oauth.js [hunk 2]
```diff
@@ -50,8 +51,20 @@ async function main() {
     if (doc.exists) {
       const data = doc.data();
       if (data && data.oauth_clients && data.oauth_clients[CLIENT_ID]) {
-        // Record is already in the database
-        continue;
+        if (REMOVE_CLIENT) {
+          // Remove this entry
+          await document.set(
+            {
```
Comment (jrgm) [inline packages/fxa-auth-server/bin/update_firestore_from_oauth.js:57]: So, did you want to test for case, and leave it alone if it is not uppercase?

File: packages/fxa-auth-server/bin/update_firestore_from_oauth.js (hunk 1 above)
Comment (bbangert) [reply packages/fxa-auth-server/bin/update_firestore_from_oauth.js:23]: I can't right now, or you won't be able to fix the ones with the error.

File: packages/fxa-auth-server/bin/update_firestore_from_oauth.js (hunk 2 above)
Comment (bbangert) [reply packages/fxa-auth-server/bin/update_firestore_from_oauth.js:57]: We know its already case sensitive, so this test is already also case sensitive. Building more fault detection in feels less useful as once this script is run and we get the appropriate results in prod, its going to be removed. This is a one-off script.

Comment (jrgm) [reply packages/fxa-auth-server/bin/update_firestore_from_oauth.js:57]: Or is the intent to run with the uppercase CLIENT_ID value and REMOVE_CLIENT=true. If it finds a match using the uppercase, then nuke it.

Comment (bbangert) [reply packages/fxa-auth-server/bin/update_firestore_from_oauth.js:57]: Yes, that was the intent. Use the upper-case version with REMOVE_CLIENT.

Comment (jrgm) [reply packages/fxa-auth-server/bin/update_firestore_from_oauth.js:57]: Okay. Thanks.

**Pull request merged**